.github/workflows/pretest-and-test.yml

```
@@ -41,3 +50,18 @@ jobs:
         pushd example
         mypy . --config-file ../setup.cfg --allow-untyped-defs --allow-untyped-calls
         popd
+
+    - name: Install for pytest
+      run: |
+        pip install -U pip wheel
+        pip install 'torch==${{ matrix.torch }}' torchvision --extra-index-url https://download.pytorch.org/whl/cpu
+        pip install pytest
+        pip install matplotlib tensorboard ipykernel ipython ipywidgets pandas optuna onnx onnxruntime pytorch-ignite
+        pip install -v -e .
+        pip list -v
+        # Test PPE is importable with pytest dependency
+        python -c 'import pytorch_pfn_extras'
```

I think most things are already done in the Install section above, so how about doing this instead?

```suggestion
        pip install -v -e '.[test]'
        pip list -v
```

---

Thank you for the suggestion!
Additional package installation is still needed  following seems more optimal:
```suggestion
        pip install matplotlib tensorboard ipython ipywidgets pandas optuna onnx pytorch-ignite
        pip install -v -e '.[test]'
        pip list -v
        # Test PPE is importable with pytest dependency
        python -c 'import pytorch_pfn_extras'
```
I still want the import only test to know ppe loads correctly with it.
And removing `ipykernel` from installation since the broken dependencies fixed somewhere